eslintrc.json

```diff
@@ -12,6 +12,17 @@
       "^[A-Z]?[a-z]+(?:[A-Z][a-z]+)*(\\.[a-z]+)*$",
       false
     ],
+    "import/no-anonymous-default-export": [
+      "error",
+      {
+        "allowAnonymousClass": false,
+        "allowAnonymousFunction": true,
```

Comment:
Shall this be `true`:

```js
"allowAnonymousFunction": true,
```

So that your editor/typechecker better knows where each function is located.

Like, this would be preffered:

```js
const myFunction = () => {};

export default myFunction;
```

or this:

```js
export default myFunction () {}
```

vs no name which is bad, because your dev toolls don't know the name of the function (and debugger):

```js
export default () => {}
```